interface/wx/font.h

```diff
@@ -322,9 +322,14 @@ class wxFontInfo
     /**
         Constructor setting the font size in points to use.
 
+        Until wxWidgets 3.1.2, the type of @a pointSize argument could only be
```

Comment:
Wouldn't it be easier to read the "current behavior" first? Maybe something like

> The type of @ a pointSize argument can be either integer or fractional, starting with wxWidgets 3.1.2. Previously it could only be @c int.

Reply:
I've ended up with this:

        The canonical type of @a pointSize argument is @c float, however any
        other integer type, as well as @c double, is also accepted for
        compatibility.

        Notice that until wxWidgets 3.1.2, the type could only be @c int.

which, I think, is better than either version.